jinahub/indexers/query/compound/NumpyPostgresQueryIndexer/manifest.yml

```diff
@@ -0,0 +1,12 @@
+manifest_version: 1
+name: NumpyFileQueryIndexer
+kind: indexer
+description: A compounded file and numpy based query indexer
+author: Jina AI Dev-Team (dev-team@jina.ai)
+url: https://jina.ai
+vendor: Jina AI Limited
+documentation: https://github.com/jina-ai/jina-hub
+version: 0.0.1
+license: apache-2.0
+keywords: [database, compound, indexer, NumpyIndexer, FileQueryIndexer]
+type: pod
```

Comment:
Is this in sync with the current hubble planning?

Reply:
No. Is there a standard defined already?